.github/workflows/rust.yml

```diff
@@ -68,3 +68,35 @@ jobs:
           push: true
           tags: ghcr.io/kubecfg/kubit:latest
         if: github.event_name != 'pull_request'
+
+  release:
```

Comment:
Looks like this change is causing release to run on every branch, not just on tags

Reply:
Yeah, I was missing the conditional bit.

I think you added that comment right as I pushed it though